### PR TITLE
OSDOCS-4825: Copy edits and structural tweaks for MicroShift 4.12 docset

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
This PR is purely editorial in nature: it makes copy edits such as fixing footnotes, numbering and capitalization, removing xrefs from a module, adding lead-in text to procedures, and retitling a couple of modules to accurately reflect their contents. Note that this docset did not publish with OCP 4.12, as the product release was held. 

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSDOCS-4825

Link to docs preview:
reorg a few assemblies (move from config to troubleshooting)
https://54793--docspreview.netlify.app/microshift/latest/microshift_troubleshooting/microshift-version.html
https://54793--docspreview.netlify.app/microshift/latest/microshift_troubleshooting/microshift-things-to-know.html
https://54793--docspreview.netlify.app/microshift/latest/microshift_troubleshooting/microshift-troubleshooting.html
https://54793--docspreview.netlify.app/microshift/latest/microshift_configuring/microshift-using-config-tools.html
https://54793--docspreview.netlify.app/microshift/latest/microshift_networking/microshift-networking.html#microshift-config-OVN-K_microshift-networking
https://54793--docspreview.netlify.app/microshift/latest/microshift_storage/persistent_storage_microshift/microshift-storage-plugin-overview.html#lvms-deployment

QE /SME reviews:
Not needed

### CODE REVIEW

File: modules/microshift-lvms-configuring.adoc [hunk 1]
```diff
@@ -32,28 +32,25 @@ device-classes: <2>
...
+<6> Boolean. Indicates that the `device-class` is used by default. Defaults to `false`.
 <7> unit. The number of stripes in the logical volume.
-<8> String. The amount of data that is written to one device before moving to the next device. 
-<9> String. Extra arguments to pas `lvcreate`, for example, `[--type=raid1"`]. 
-+
-[WARNING]
+<8> String. The amount of data that is written to one device before moving to the next device.
+<9> String. Extra arguments to pas `lvcreate`, for example, `[--type=raid1"`].
+
+[IMPORTANT]
```
Comment (JoeAldinger) [inline modules/microshift-lvms-configuring.adoc:45]: I'm not sure I fully understand the consequence explained in the admonition, but it seems to imply that data loss could possibly occur. If so, I think WARNING is correct: https://redhat-documentation.github.io/supplementary-style-guide/#conversational-style

File: modules/microshift-lvms-system-requirements.adoc [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
 
 :_content-type: CONCEPT
-[id="lvms-system-requirements"]
+[id="lvms-system-requirements_{context}"]
 = LVMS system requirements
 
 {product-title}'s LVMS requires the following system specifications.
 
-[id="lvms-volume-group-name"]
+[id="lvms-volume-group-name_{context}"]
```
Comment (JoeAldinger) [inline modules/microshift-lvms-system-requirements.adoc:11]: These headings look _italicized_ but they aren't. Not sure.

**Review by JoeAldinger — COMMENTED** [review]
/remove-label peer-review-in-progress
/label peer-review-done

File: modules/microshift-firewall-allow-traffic.adoc [hunk 3]
```diff
@@ -32,7 +36,11 @@ $ sudo firewall-offline-cmd --permanent --zone=trusted --add-source=169.254.169.
 
 [id="microshift-firewall-applying-settings_{context}"]
 == Applying firewall settings
-After you have finished configuring, run the following command to restart the firewall and apply settings:
+
+To apply firewall settings, use the following one-step procedure:
+
+.Procedure
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply modules/microshift-firewall-allow-traffic.adoc:42]: Yeah, had some discussion about whether this was a procedure at all, or just part of the previous. Going with modularizing.

**Review by skrthomas — COMMENTED** [review]
One small capitalization question Kickstart vs kickstart in 2 links.

File: microshift_install/microshift-embed-in-rpm-ostree.adoc [hunk 4]
```diff
@@ -54,8 +54,8 @@ include::modules/microshift-provisioning-ostree.adoc[leveloffset=+1]
 . xref:../microshift_install/microshift-install-rpm.adoc#system-requirements-installing-microshift[System requirements for installing {product-title}].
 . Red Hat Hybrid Cloud Console link:https://console.redhat.com/openshift/install/pull-secret[pull secret].
 . xref:../microshift_networking/microshift-networking.adoc#microshift-firewall-req-settings_microshift-networking[Required firewall settings].
-. link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_an_advanced_rhel_8_installation/creating-kickstart-files_installing-rhel-as-an-experienced-user[Creating a kickstart file]
-. link:https://access.redhat.com/solutions/60959[How to embed a Kickstart file into an ISO image]
+. link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_an_advanced_rhel_8_installation/creating-kickstart-files_installing-rhel-as-an-experienced-user[Creating a kickstart file].
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply microshift_install/microshift-embed-in-rpm-ostree.adoc]: Yep--great catch! RHEL caps it, so I will here. Picked it up in the module as well.

**Pull request merged**